src/Core/ActionBar/ActionsBarButton.php

```diff
@@ -24,7 +24,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-namespace PrestaShopBundle\Component\ActionBar;
+namespace PrestaShop\PrestaShop\Core\ActionBar;
```

Comment:
I suggest `PrestaShop\PrestaShop\Core\Action` as the namespace, to not limit it to only toolbar buttons This namespace could be enriched later with row actions, grid actions, and so on this way Instead of creating independent namespaces for each type of action